airbyte-webapp/src/views/Connector/ConnectorCard/ConnectorCard.tsx

```diff
@@ -90,7 +90,8 @@ export const ConnectorCard: React.FC<ConnectorCardCreateProps | ConnectorCardEdi
   const [isFormSubmitting, setIsFormSubmitting] = useState(false);
   const [logsVisible, setLogsVisible] = useState(false);
 
-  const { setDocumentationUrl, setDocumentationPanelOpen } = useDocumentationPanelContext();
+  const { setDocumentationUrl, setDocumentationPanelOpen, setSelectedConnectorDefinition } =
```

Comment:
ATM, we need to add information about the connectorDefinition to check if it's a source, but we'll use this more extensively in the next PR, as we need this information to actually fetch the schema of the selected source